### DISCUSSION TITLE
Makefile License

### DISCUSSION THREAD

**Discussion by Michael2MacDonald** [luni64/VisualTeensy#66] [Q&A] [answered]
Are the generated makefiles also under the MIT license or are they unlicensed?

**Comment by luni64** [comment]
You can use them for whatever you want but I don't take any responsibility for them.  I added some information to the licence. 

May I ask what your are planning to do with the makefiles?

**Reply by Michael2MacDonald** [reply]
I am working on a project and might include the makefile in the GitHub repo so I wanted to know if I have to include a license notice at top of the file or not.

**Comment by luni64** [comment, marked as answer]
You can definitely add them to your repo. Users might need to adjust paths of course.
